### DISCUSSION TITLE
[Question] Why two artifacts angus-mail and jakarta.mail?

### DISCUSSION THREAD

**Discussion by space88man** [eclipse-ee4j/angus-mail#82] [Q&A] [answered]
Both jakarta.mail and angus-mail are module-info.java-only  that refer to other the packages. The two module-info files look very similar.

Could you clarify the relationship between the two artifacts?

Confusion from dependencies in the wild:
* Spring mail integration refers to jakarta.mail
* Apache CXF refers to angus-mail

Which is the "correct" one to use?
* compile with `jakarta.mail-api:2.1.1` and package with `angus-mail` (Apache CXF 4.0+ )
* compile and package with `jakarta.mail` (Spring 6.0+, Spring Boot 3.0+ )

**Comment by lukasj** [comment, marked as answer]
depends on the use-case and preferences and [the web](https://eclipse-ee4j.github.io/angus-mail/#Download_Angus_Mail_Release) already lists and describes all options.

 In short: `jakarta.mail:jakarta.mail-api` + `org.eclipse.angus:angus-mail` = `org.eclipse.angus:jakarta.mail`
